app/src/main/java/org/jellyfin/androidtv/playback/overlay/actions/AdjustAudioDelayAction.java

```diff
@@ -0,0 +1,14 @@
+package org.jellyfin.androidtv.playback.overlay.actions;
+
+import android.content.Context;
+
+import org.jellyfin.androidtv.R;
+import org.jellyfin.androidtv.playback.overlay.CustomPlaybackTransportControlGlue;
+
+public class AdjustAudioDelayAction extends CustomAction {
```

Comment:
A lot of actions have the exact same code with only the name and icon changed. Wouldn't it be better to create an `IconAction` which accepts the icon as parameter in the constructor? This way you can re-use that class for all simple actions.

Maybe even add it to the `CustomAction` class so you don't even need to extend it.